packages/calcite-components/src/components/combobox/combobox.tsx

```diff
@@ -741,8 +749,8 @@ export class Combobox
   }
 
   private getValue(): string | string[] {
-    const items = this.selectedItems.map((item) => item?.value?.toString());
-    return items?.length ? (items.length > 1 ? items : items[0]) : "";
+    const items = this.selectedItems.map((item) => item.value?.toString());
```

Comment:
This `.toString()` reminds me of #11370 where we may want to consider changing value from `any` to `string`.